framework/test/mocha/unit/modules/http_api/init_steps/bootstrap_storage.js

```diff
@@ -26,6 +26,7 @@ describe('init_steps/bootstrap_storage', () => {
 			components: {
 				storage: {
 					bootstrap: sinonSandbox.stub().resolves(true),
+					registerEntity: sinonSandbox.stub(),
```

Comment:
I'll not force to fix it in this PR but the way we stub the storage component is wrong here. Stubbing entity methods are fine but registerEntity is not interacting with I/O and there is no need to fake it. I just wanted to raise it. You can skip it in this PR.